packages/@blockprotocol/graph/src/internal/mutate-subgraph/is-equal.ts

```diff
@@ -0,0 +1,67 @@
+export const isEqual = (first: any, second: any): boolean => {
```

Comment:
Is this from somewhere? Do we want to attribute it? (In case there are updates there or someone discovers limitations)

Reply:
I see this was actually moved from an existing file, did I add it or did you? 😅 

My question still stands if we do know where it's from

Reply:
You added it – I think it was probably from a gist that I sent you. I can check if you adopted it and link it

Reply:
Oops, please do

Reply:
https://github.com/blockprotocol/blockprotocol/pull/869/commits/d049a78aa2e9a55bc48ca7d28a5817ca0130e765